developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_29.rst

```diff
@@ -52,7 +52,7 @@ Back-end changes
 Added APIs
 ^^^^^^^^^^
 
-* tbd
+* ``OCP\IURLGenerator::URL_REGEX_NO_MODIFIERS``: Changed to match localhost and hostnames with ports.
```

Comment:
Move to `Changed APIs` ? :P

Reply:
Ouch, nice catch :grimacing: Changed now.